doc/source/whatsnew/v0.16.1.txt

```diff
@@ -16,8 +16,41 @@ We recommend that all users upgrade to this version.
 
 Enhancements
 ~~~~~~~~~~~~
+.. _whatsnew_0161.enhancements.sample:
 
+Sample
+^^^^^^^^^^^^^^^^
 
+Series, DataFrames, and Panels now have a new method: :meth:`~pandas.core.sample`.
```

Comment:
the `pandas.core.sample` should be `pandas.Series.sample` (or with DataFrame), as this is how it is exposed publicly.